### PR TITLE
gmoccapy: fix source typo

### PR DESCRIPTION
**Pull request by luzpaz**
Fix source typo `responce` to `response`

### CODE REVIEW

File: src/emc/usr_intf/gmoccapy/gmoccapy.py [hunk 1]
```diff
@@ -2188,16 +2188,16 @@ def _show_user_message(self, pin, message):
...
         elif message[1] == "yesnodialog":
             if pin.get():
                 self.halcomp["messages." + message[2] + "-waiting"] = 1
                 self.halcomp["messages." + message[2] + "-response"] = 0
                 title = "Pin " + message[2] + " message"
-                responce = self.dialogs.yesno_dialog(self, message[0], title)
+                response = self.dialogs.yesno_dialog(self, message[0], title)
                 self.halcomp["messages." + message[2] + "-waiting"] = 0
-                self.halcomp["messages." + message[2] + "-response"] = responce
+                self.halcomp["messages." + message[2] + "-response"] = response
```
Comment (luzpaz) [inline src/emc/usr_intf/gmoccapy/gmoccapy.py:2200]: this is probably wrong. Can we rename the variable something other than a typo of `response` ?

Comment (hansu) [reply src/emc/usr_intf/gmoccapy/gmoccapy.py:2200]: No, I think removing the typo is not wrong. Why do you think so?

By the way, I have also done these changes (in an open PR): https://github.com/LinuxCNC/linuxcnc/pull/1629/commits/7bf7745e894953afd64c90043e5ff720c3aba8d4 :smile:

Comment (luzpaz) [reply src/emc/usr_intf/gmoccapy/gmoccapy.py:2200]: So this PR is obsolete in that case?

Comment (hansu) [reply src/emc/usr_intf/gmoccapy/gmoccapy.py:2200]: Yeah kind of. Sorry.

**Pull request closed without merging**